Gruntfile.js

```diff
@@ -333,6 +370,8 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-version');
   grunt.loadNpmTasks('grunt-tagrelease');
   grunt.loadNpmTasks('chg');
+  grunt.loadNpmTasks('grunt-fastly');
+  grunt.loadNpmTasks('grunt-github-releaser');
```

Comment:
we should just get [load-grunt-tasks](https://www.npmjs.com/package/load-grunt-tasks)